src/index.js

```diff
@@ -17,10 +16,9 @@ root.render(
       </BrowserRouter>
     </Provider>
   </React.StrictMode>,
-  document.getElementById('root'),
 );
 
 // If you want to start measuring performance in your app, pass a function
 // to log results (for example: reportWebVitals(console.log))
 // or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
```

Comment:
- [ ] It would be nice if you remove the commented lines as suggested by the previous reviewer to make your codebase clean 👍